lib/validators/specTransformer.ts

```diff
@@ -18,7 +19,23 @@ export interface SchemaObjectInfo {
   readonly infoFunc: InfoFunc
 }
 
-export const getSchemaObjectInfo = (model: SchemaObject): SchemaObjectInfo | undefined => {
+export const getTitle = (model: json.JsonObject|json.JsonArray) => {
+  const infoFunc = getInfoFunc(model)
+  if (infoFunc === undefined) {
+    return undefined
+  }
+  const info = infoFunc()
+  return {
+    path: getPath(info),
+    position: info.position,
+    url: getRootObjectInfo(info).url,
+    directives: getAllDirectives(model, []),
```

Comment:
is this title a known format ? can we type it ?

Reply:
i see there is `TitleObject`

---
In reply to: [251953142](https://github.com/Azure/oav/pull/382#discussion_r251953142) [](ancestors = 251953142)